x/concentrated-liquidity/incentives_test.go

```diff
@@ -717,7 +717,13 @@ func (s *KeeperTestSuite) TestCalcAccruedIncentivesForAccum() {
 	}
 }
 
+// Testing strategy:
+// 1. Create a position
+// 2. Let a fixed amount of time pass, enough to qualify it for some (but not all) uptimes
+// 3. Let a variable amount of time pass determined by the test case
```

Comment:
It seems the variable amount of time in every test case is one hour, but I think we can expand this in a future iteration